Settings-test.toml

```diff
@@ -9,6 +9,11 @@ password = "geoengine"
 [operators.gdal_source]
 raster_data_root_path = "../test_data/raster" # relative to sub crate directory for tests
 
+[logging]
+# Minimum log level. Can be one of error, warn, info, debug, trace
+# or a more detailed spec. See https://docs.rs/flexi_logger/0.17.1/flexi_logger/struct.LogSpecification.html.
+log_spec = "debug"
```

Comment:
Do we need `debug` logging here or is this just for own debug purposes?

Reply:
Was for local testing. Removed it. 8fb81dbed4ec76c5ed14d5584083f7938818d7fe